CHANGES.md

```diff
@@ -22,7 +21,7 @@ OpenSSL Releases
 OpenSSL 3.0
 -----------
 
-### Changes between 1.1.1 and 3.0 [xx XXX xxxx] ###
+### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
```

Comment:
@mattcaswell does this change affect the regex of the release script?

Reply:
I was going to look at that next.